### PR TITLE
system/mongodb: Lower GCC version

### PR DESCRIPTION
**Pull request by willysr**

### CODE REVIEW

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/mongodb

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/mongodb

**Comment by willysr** [comment]
@sbo-bot: build python/python3-bson

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/mongodb

**Comment by willysr** [comment]
@sbo-bot: build x86_64 python/pymongo

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/mongodb

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/mongodb

**Comment by aclemons** [comment]
```
Building system/mongodb (update for version 7.0.4) -------------------- 10:04:49
Testing SlackBuild files ... done.
Downloading source files ...
Verifying source files ... done.
mount: /tmp/slackrepo.Md1nGe/chrootdir/root/.Xauthority: special device /root/.Xauthority does not exist.
Installing dependencies ...
snappy-1.1.8-x86_64-2_SBo: A fast compressor/decompressor .............. [ 110K]
mongo-c-driver-1.25.1-x86_64-2_SBo: C driver for MongoDB ............... [ 2.4M]
yaml-cpp-0.6.3-x86_64-1_SBo: YAML parser ............................... [ 1.1M]
cheetah3-3.3.2-x86_64-1_SBo: Python3 template engine and code-generation [ 1.4M]
python-zipp-3.8.0-x86_64-2_SBo: pathlib-compatible Zipfile object wrappe [  50K]
python-importlib_metadata-4.10.1-x86_64-3_SBo: Python package metadata . [ 130K]
python3-flit_core-3.9.0-x86_64-1_SBo: Simplified pkging of Python module [ 140K]
python3-installer-0.7.0-x86_64-2_SBo: Library for installing Python pack [ 1.1M]
python3-pyproject-hooks-1.0.0-x86_64-4_SBo: Wrappers to call PEP 517 bui [ 110K]
python3-build-1.0.3-x86_64-3_SBo: a simple, correct PEP517 package build [ 180K]
python3-wheel-0.41.3-x86_64-4_SBo: A built-package format for Python ... [ 610K]
python3-setuptools-opt-65.1.1-x86_64-5_SBo: Install setuptools from -cur [ 8.7M]
pymongo-4.6.1-x86_64-3_SBo: Python Driver/Tools for mongodb ............ [ 4.2M]
python3-attrs-21.4.0-x86_64-1_SBo: Classes Without Boilerplate ......... [ 630K]
python3-editables-0.5-x86_64-5_SBo: A Python library for creating editab [  50K]
python3-pathspec-0.11.2-x86_64-2_SBo: Library for gitignore style matchi [ 250K]
python3-pluggy-1.3.0-x86_64-2_SBo: mechanisms for python ............... [ 160K]
python3-calver-2022.6.26-x86_64-11_SBo: setuptools extension for CalVer  [  60K]
python3-trove-classifiers-2023.11.22-x86_64-1_SBo: Canonical source for  [ 170K]
python3-hatchling-1.18.0-x86_64-8_SBo: Hatch build backend ............. [ 940K]
python3-hatch_fancy_pypi_readme-23.1.0-x86_64-2_SBo: Hatch metadata plug [ 110K]
python3-hatch_vcs-0.3.0-x86_64-12_SBo: Hatch plugin for versioning with  [  80K]
python3-pyrsistent-0.19.3-x86_64-2_SBo: persistent collections for Pytho [ 460K]
python3-jsonschema-4.17.3-x86_64-7_SBo: Implementation of JSON Schema .. [ 1.1M]
python3-psutil-5.9.6-x86_64-3_SBo: python interface for process and syst [ 2.6M]
snowballstemmer-2.2.0-x86_64-1_SBo: 16 stemmer algorithms for python ... [ 2.7M]
Adding groups and users:
  groupadd -g 285 mongo;
  useradd  -u 285 -g mongo -c mongodb -d /var/lib/mongodb -s /bin/false  mongo;
Running mongodb.SlackBuild ...                                      ETA 13:36:??
env MAKEFLAGS='-j24 -l13' nice -n 5 bash ./mongodb.SlackBuild
```

Looks like it will take 3.5 hours to build - it's only building with one core though. Can we tell the build to use all cores in some way?

File: system/mongodb/mongodb.SlackBuild [hunk 1]
```diff
@@ -101,6 +101,9 @@ find -L . \
 # thanks gentoo, also for these patches
 for i in $CWD/patches/* ; do patch -p1 < $i ; done
```
Comment (aclemons) [inline system/mongodb/mongodb.SlackBuild:103]: I think we should use the misc/rmlint/rmlint.SlackBuild scons handling as an example

We can pull the job counter from MAKEFLAGS and this should be many time faster building.

Comment (willysr) [reply system/mongodb/mongodb.SlackBuild:102]: Done

**Comment by willysr** [comment]
@sbo-bot: build x86_64 system/mongodb

**Comment by willysr** [comment]
@aclemons i need your help to setup hints for this :)

**Comment by aclemons** [comment]
@sbo-bot: build x86_64 system/mongodb

**Comment by aclemons** [comment]
mongo

Looking much better - hopefully won't take 7 hours this time.

**Comment by willysr** [comment]
oh wow it cuts down to 1h only

**Pull request closed without merging**